Project/core/Log.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project
+{
+    public enum Type { Create_Reserwation, Check_In, Check_Out, Extend_Reservations, Canceled_Reservations,  Cleaning, Create_Room, Create_Client, Create_Emplyee, Restor_Canceled_Reservation, Other }
```

Comment:
Nazwa type nic nie mówi, raczej LogType
```suggestion
    public enum LogType { Create_Reserwation, Check_In, Check_Out, Extend_Reservations, Canceled_Reservations,  Cleaning, Create_Room, Create_Client, Create_Emplyee, Restor_Canceled_Reservation, Other }
```